README.md

```diff
@@ -1,48 +1,57 @@
-# Modern python app boilerplate
+# asyncstatsd
 
-Included in this boilerplate:
+## Installation
 
- - pyenv for python version management
- - [poetry](https://python-poetry.org/) for dependency management integrated with pyenv
- - [pre-commit](https://pre-commit.com/) for linting and formatting
- - [pytest](https://docs.pytest.org/en/stable/) for testing
- - [black]() for formatting
- - [flake8]() for linting
- - [isort]() for sorting imports
- - [mypy]() for static type checking
- - virtualenv created by poetry resides in `.venv` folder
+    $ pip install asyncstatsd
 
+## Usage
 
-## How to start
+### Pure statsd
```

Comment:
similar (minor): nice to see known compatible statsd / Datadog version(s),

Reply:
Yes, I'll add brief discussion which explains why I decided to write this